src/components/highRiskMap.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react"
+
+import ReactDom from "react-dom"
+import PropTypes from "prop-types"
+import L from "leaflet"
+import "leaflet.markercluster"
+import "leaflet.markercluster/dist/MarkerCluster.css"
+import "leaflet.markercluster/dist/MarkerCluster.Default.css"
+import "leaflet/dist/leaflet.css"
+// import mapMarker from "./icons/map-marker.png"
+
+// const icons = {
+//   defaultMarker: L.icon({
+//     iconUrl: mapMarker,
+//     iconSize: [30, 30],
+//     iconAnchor: [15, 30],
+//     popupAnchor: [0, -30],
+//   }),
+// }
+const limit = 1.5
+export default class highRiskMap extends Component {
```

Comment:
in the future please consider using functional components with react hook for consistency 😉